include/seqan3/range/views/minimiser.hpp

```diff
@@ -332,57 +324,60 @@ class minimiser_view<urng_t>::window_iterator
 
 private:
     //!brief Iterator to last element in range.
-    sentinel_t urange_end;
+    urng_sentinel_t urng_sentinel;
 
     //!\brief The minimiser value.
     value_type minimiser_value{};
 
     //!\brief Iterator to the rightmost value of one window.
-    it_t window_right;
+    urng_iterator_t urng_iterator;
```

Comment:
```suggestion
    urng_iterator_t urng_iterator{};
```

Reply:
This didn't work?

Reply:
I applied these suggestions, but forgot to pull them into my local branch. I rewrote the git history (without these suggestions). Sorry wasn't on purpose.